src/samples/EducationStart/ClaimPage.tsx

```diff
@@ -0,0 +1,245 @@
+    // PCore.getMashupApi().createCase('HMRC-ChB-Work-HICBCPreference', PCore.getConstants().APP.APP);
+    PCore.getMashupApi().createCase('HMRC-ChB-Work-EducationStart', PCore.getConstants().APP.APP);
+
+  };
+
+  /* ***
+   * Application specific PCore subscriptions
+   *
+   * TODO Can this be made into a tidy helper? including its own clean up? A custom hook perhaps
+   */  
+
+  // And clean up
+
+  useEffect(() => {
+    document.addEventListener('SdkConstellationReady', () => {
+      PCore.onPCoreReady(() => {
+        if(!pCoreReady){
+          setPCoreReady(true);
+          PCore.getPubSubUtils().subscribe(
+            PCore.getConstants().PUB_SUB_EVENTS.CONTAINER_EVENTS.CLOSE_CONTAINER_ITEM,
+            () => {
+              // console.log("SUBEVENT!!! showStartPageOnCloseContainerItem")
+              setShowPega(false);
+            },
+            'showStartPageOnCloseContainerItem'
+          );
+          startClaim();
+        }
+      });
+      settingTimer();
+    });
+    
+    return () => {
+      PCore.getPubSubUtils().unsubscribe(
+        PCore.getConstants().PUB_SUB_EVENTS.CONTAINER_EVENTS.CLOSE_CONTAINER_ITEM,
+        'showStartPageOnCloseContainerItem'
+      );
+    };
+  }, []);
+
+    /* ***
+     * Application specific PCore subscriptions
+     * 
+     * TODO Can this be made into a tidy helper? including its own clean up? A custom hook perhaps 
+     */
+    document.addEventListener('SdkConstellationReady', () => {
+        PCore.onPCoreReady(() => {        
+        PCore.getPubSubUtils().subscribe(
+            PCore.getConstants().PUB_SUB_EVENTS.CONTAINER_EVENTS.CLOSE_CONTAINER_ITEM,
+            () => {
+                // console.log("SUBEVENT!!! showStartPageOnCloseContainerItem")
+                setShowPega(false);
+            },
+            'showStartPageOnCloseContainerItem'
+        );
+        })
+        settingTimer();
+        PCore.getStore().subscribe(() => staySignedIn(setShowTimeoutModal, '', null, true, false));
+    }); 
+    
+    // And clean up
+
+    useEffect(() => {
+        return () => {
+            PCore.getPubSubUtils().unsubscribe(PCore.getConstants().PUB_SUB_EVENTS.CONTAINER_EVENTS.CLOSE_CONTAINER_ITEM,'showStartPageOnCloseContainerItem')
+        }
+    }, [])
+
+
+    return ( <>
+      <TimeoutPopup
+        show={showTimeoutModal}
+        staySignedinHandler={() =>
+          staySignedIn(setShowTimeoutModal, 'D_ClaimantWorkAssignmentChBCases')
+        }
+        signoutHandler={triggerLogout}
+        isAuthorised={false}  
+        signoutButtonText="Sign out"
+        staySignedInButtonText="Stay signed in"
+      >
+        <h1 id="hmrc-timeout-heading" className="govuk-heading-m push--top">You’re about to be signed out</h1>
+        <p className="govuk-body hmrc-timeout-dialog__message" aria-hidden="true">For your security, we will sign you out in <span id="hmrc-timeout-countdown" className="hmrc-timeout-dialog__countdown">2 minutes</span>.</p>
```

Comment:
use the text content from the language json

Reply:
This page will be developed once the userstory will be ready. This is just a dummy content has been used at the moment.

Reply:
Added content in the language json